mappings/net/minecraft/util/SortedSet.mapping

```diff
@@ -0,0 +1,30 @@
+CLASS net/minecraft/class_4706 net/minecraft/util/SortedSet
```

Comment:
This name is to similar to that of `java.util.SortedSet`. :thinking:

Reply:
Didn't know that existed... It does act like `java.util.SortedSet`, except it has fewer methods. A possible name could be `SortedArraySet`, though that might make people think it's an implementation of `SortedSet`.

Any other suggestions @liach ?

Reply:
looks great to me.